ZeroWasteFriendly/Scenes/Library/LibraryCarouselCollectionViewCell/LibraryCarouselCollectionViewCell.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  LibraryCarouselCollectionViewCell.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 08/03/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+final class LibraryCarouselCollectionViewCell: UICollectionViewCell {}
```

Comment:
I am used to remove 'CollectionView' and 'TableView' from cell names, as it is mostly groupped properly in project groups and there is no need for such a long names

LibraryCarouselCollectionViewCell -> LibraryCarouselCell